src/main/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilder.java

```diff
@@ -64,7 +64,16 @@ FilterVariable generateFilter(SnapshotBuilderProgramDataRangeCriteria rangeCrite
                 new Literal(rangeCriteria.getHigh()))));
   }
 
+   FilterVariable selectAllValuesOfListCriteria() {
+    return platform -> "1=1";
+  }
+
   FilterVariable generateFilter(SnapshotBuilderProgramDataListCriteria listCriteria) {
+
+    var listCriteriaValueIsEmpty = listCriteria.getValues().isEmpty();
```

Comment:
Instead of leaving a comment, "is listCriteria Value Empty" or letting readers interpret
```listCriteria.getValues().isEmpty()```
I wanted to just make a variable. Thoughts?